### PR TITLE
MODELIX-718 Store Git metadata when syncing

### PR DESCRIPTION
**Pull request by mhuster23**
This PR adds the option for users to set arbitrary custom properties for the rootNode during the sync.
For example, users could store git metadata this way.

### CODE REVIEW

File: bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/tasks/ImportIntoModelServer.kt [hunk 1]
```diff
@@ -91,6 +96,11 @@ abstract class ImportIntoModelServer @Inject constructor(of: ObjectFactory) : De
                     logger.info("Got root node: {}", rootNode)
                     logger.info("Calculating diff...")
                     ModelImporter(rootNode, continueOnError.get()).importFilesAsRootChildren(files)
+
+                    logger.info("Setting meta properties...")
+                    for ((key, value) in metaProperties.get()) {
+                        rootNode.setPropertyValue(IProperty.fromName(key), value)
+                    }
```
Comment (languitar) [inline bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/tasks/ImportIntoModelServer.kt:103]: Are there any reserved properties already on the root node the plugin implementation should shield against from blindly accepting them?

File: bulk-model-sync-gradle-test/ci.sh
Comment (languitar) [inline bulk-model-sync-gradle-test/ci.sh:1]: What's the essence of the changes here? Pretty hard to understand. And why could the test run before?

**Review by languitar — DISMISSED** [review]
Generally fine.

It's a bit hard to understand the changes to the test CI bash script and the reasons for the performed refactorings to the tests. A few more explanations in the commit messages would make these changes easier to understand and to reason about.

File: bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/tasks/ImportIntoModelServer.kt (hunk 1 above)
Comment (mhuster23) [reply bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/tasks/ImportIntoModelServer.kt:103]: As far as I know, the root node doesn't contain any properties by default.

File: bulk-model-sync-gradle-test/ci.sh
Comment (mhuster23) [reply bulk-model-sync-gradle-test/ci.sh:1]: It depended on how you invoked the script. The old version used relative paths. The new one uses absolute paths.

**Comment by mhuster23** [comment]
Reworded some commits to improve clarity.

**Review by languitar — APPROVED** [review]

**Pull request merged**